desktop/core/src/desktop/conf.py

```diff
@@ -1071,7 +1071,7 @@ def get_deprecated_login_lock_out_by_combination_browser_user_agent():
   help=_("Configuration options for user authentication into the web application."),
   members=dict(
     BACKEND=Config("backend",
-                   default=["desktop.auth.backend.AllowFirstUserDjangoBackend"],
+                   default=['axes.backends.AxesBackend', "desktop.auth.backend.AllowFirstUserDjangoBackend"],
```

Comment:
Won't this brake when setting manually, e.g. LDAP, backends?

Can't we auto add it in settings.py?